voltixApp/VoltixApp/Views/Keysign/KeysignDiscoveryView.swift

```diff
@@ -179,45 +133,24 @@ struct KeysignDiscoveryView: View {
     
     var bottomButtons: some View {
         Button(action: {
-            self.startKeysign(allParticipants: self.selections.map { $0 })
-            self.currentState = .Keysign
-            self.participantDiscovery.stop()
+            self.viewModel.startKeysign()
```

Comment:
You can remove self when referencing the view models. 
Same goes for other classes as well.

Reply:
yeah, I know , I am using swiftformat , it will automatically add it back for me , so I will keep it.